src/entities/PersistentSignedOrderEntity.ts

```diff
@@ -0,0 +1,93 @@
+import { OrderEventEndState } from '@0x/mesh-rpc-client';
```

Comment:
Given that we have `synchronize` disabled in production I think we'd need to create a migration for this Entity in order for it to take effect in prod, or am I missing something? 🤔

Reply:
We used `synchronize` to add the last table and it was meh. I'll create a migration file.